controllers/volumereplicationgroup_controller.go

```diff
@@ -861,12 +861,16 @@ func (v *VRGInstance) updateVRGStatus(updateConditions bool) (bool, error) {
 			return true, fmt.Errorf("failed to update VRG status (%s/%s)", v.instance.Name, v.instance.Namespace)
 		}
 
-		v.log.Info(fmt.Sprintf("Updated VRG Status %+v", v.instance.Status))
+		dataReadyCondition := findCondition(v.instance.Status.Conditions, VRGConditionTypeDataReady)
+		v.log.Info(fmt.Sprintf("Updated VRG Status (%s/%s), VolRep pvccount (%d), VolSync pvccount(%d)"+
+			" DataReady Condition (%s)", v.instance.Name, v.instance.Namespace,
+			len(v.volRepPVCs), len(v.volSyncPVCs), dataReadyCondition))
 
 		return !v.areRequiredConditionsReady(), nil
 	}
 
-	v.log.Info(fmt.Sprintf("Nothing to update %+v", v.instance.Status))
+	v.log.Info(fmt.Sprintf("Nothing to update (%s/%s), VolRep pvccount (%d), VolSync pvccount(%d)",
```

Comment:
This message should drop the v.instance.* as those are already keys in v.log. and would be logged.

Reply:
Done